src/block.json

```diff
@@ -0,0 +1,5 @@
+{
+	"description": "This is a bit of a hack: using a bare block.json lets us trick wp-scripts into building our frontend and editor bundles",
```

Comment:
sneaky! 

I think you can also add entry paths to wp-scripts via a command line argument, but this works.

Reply:
Tried that and it wasn't working as I'd expect, but didn't try very hard. Will have a go at it again next project :)